src/NumericalAlgorithms/Spectral/FiniteDifference.cpp

```diff
@@ -16,10 +16,12 @@ std::pair<DataVector, DataVector> compute_collocation_points_and_weights<
     Basis::FiniteDifference, Quadrature::CellCentered>(
```

Comment:
Picking a line at random: can you add tests for the changes?

Reply:
There are also issues with the existing tests... they apparently want the inverse weight function to be defined.